modules/op-configuring-retention-policy-results.adoc

```diff
@@ -0,0 +1,37 @@
+// This module is included in the following assembly:
+//
+// * cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-retention-policy-results_{context}"]
+= Configuring retention policy for {tekton-results}
+
+By default, {tekton-results} stores pipeline runs, task runs, events, and logs indefinitely. This leads to an unnecesary use of storage resources and can affect your database performance.
+
+You can configure retention policy for {tekton-results} at cluster level to remove older results and their associated records and logs. You can achieve this by editing the `TektonResult` custom resource (CR).
+
+.Prerequisites
+
+* You installed {tekton-results}.
+
+.Procedure
+
+* In the `TektonResult` CR, configure the retention policy for {tekton-results}, as shown in the following example:
```

Comment:
Right now, this procedure only shows an example of what a CR looks like with the retention policy configured. Not the steps to actually make the updates.

Should this instead show a step to update the CR and apply the changes? For example, by using `oc edit` and then adding the necessary options, and then saving the file to apply the changes? Or could use a YAML file and `oc apply` to apply the updates.